libs/providers/flagd/package.json

```diff
@@ -6,7 +6,6 @@
     "current-version": "echo $npm_package_version"
   },
   "peerDependencies": {
-    "@grpc/grpc-js": "^1.6.0",
```

Comment:
This shouldn't be a peer. I don't know how it got here, but it's automatically included in the deps by NX. This is how the automatically created dist/package.json looks:

```
  "dependencies": {
    "@grpc/grpc-js": "1.8.20",
    "@protobuf-ts/runtime-rpc": "2.9.0",
    "lru-cache": "10.0.1",
    "util": "0.12.5"
  }
```